health_check/contrib/redis/backends.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from django.conf import settings
+from kombu import Connection
```

Comment:
`kombu` is part of celery and current not listed in the requirements. So maybe it makes sense to switch to another redis client. There also should be documentation, or an `extra` requirement addtion to the setuptools config.

Reply:
Added some documentation to the README, not a whole lot, but not sure if any more is really necessary.